test/e2e/fixture/fixture.go

```diff
@@ -70,6 +70,10 @@ func getKubeConfig(configPath string, overrides clientcmd.ConfigOverrides) *rest
 // creates e2e tests fixture: ensures that Application CRD is installed, creates temporal namespace, starts repo and api server,
 // configure currently available cluster.
 func init() {
+	// this enables colors and disables full timestamps
+	log.SetFormatter(&log.TextFormatter{ForceColors: true})
```

Comment:
this changes the logging output to be ... in my mind ... nicer

Reply:
![Screen Shot 2019-07-02 at 15 24 22](https://user-images.githubusercontent.com/1142830/60550652-871c6000-9cdd-11e9-8b2d-99c4d1b54f02.png)